simulation/bombard.py

```diff
@@ -13,11 +13,11 @@
     print("python3 ./bombard.py <UUID> <AUTH_TOKEN>")
     sys.exit()
 
-num = 10
+num = 50
 uuid = sys.argv[1]
 auth = sys.argv[2]
 
-data_json_type = "JSON data"
+data_json_type = "  JSON data"
```

Comment:
Is this whitespace important for logic? Can't see any reference for this.

Reply:
no, the output just looks better

Reply:
Fair enough, I can see that this get's them to the same length. But I think this is not the perfect locatio for formatting..

would you be okay with removing the whitespace, if we padded the format string to at least 12 characters on line 112?

```diff
-             print("- - - HASH MISMATCH: {}".format(request_type))
+             print("- - - HASH MISMATCH: {0: >12}".format(request_type))
```

Reply:
line 112 is not the only place where the type is printed. this entire script is really just for me to play around and quickly test inputs, it was not meant to be any official test script. do you think it would be acceptable to not hold it to the high standards that we usually apply?

Reply:
sure, this was just an optional change, I had already approved

Reply:
thank you